instrumentation/opentelemetry-instrumentation-httpx/tests/test_httpx_integration.py

```diff
@@ -57,6 +57,10 @@ def _async_call(coro: typing.Coroutine) -> asyncio.Task:
 
 
 def _response_hook(span, request: "RequestInfo", response: "ResponseInfo"):
+    # httpx < 0.20.0 url is a tuple of len 4 and not a httpx.URL
```

Comment:
Maybe you can move this outside into its own function to avoid duplicating it for each test

Reply:
Sure, done 6da35cbe78e21c1a686cc4a1ecb3b4f032ab8079